description/urdf/plugins.gazebo.xacro

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0"?>
+<robot name="plugins" xmlns:xacro="http://ros.org/wiki/xacro">
+
+    <!-- Constants -->
+    <xacro:include filename="$(find description)/urdf/constants.xacro"/>
+
+    <!-- LIDAR -->
+    <gazebo reference="scan_link">
+        <sensor type="ray" name="gpu_hokuyo_sensor">
+            <pose>0 0 0 0 0 0</pose>
+            <!-- Set visualize to true for now -->
+            <visualize>true</visualize>
```

Comment:
only visualize for debugging, if the lidar works as expected, then set `visualize` to `False`